.github/ISSUE_TEMPLATE/1_activation.yml

```diff
@@ -0,0 +1,92 @@
+name: I am having difficulty installing Flutter or getting it to work
+description: You have run into problems while downloading or installing Flutter, or the
+  "flutter" tool is crashing, or you are running into some other issue before even
+  being able to use "flutter run".
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thank you for using Flutter!
+
+        If you are looking for support, please check out our documentation
+        or consider asking a question on Stack Overflow:
+
+          - https://flutter.dev/
+          - https://api.flutter.dev/
+          - https://stackoverflow.com/questions/tagged/flutter?sort=frequent
+
+        If you have found a bug or if our documentation doesn't have an answer
+        to what you're looking for, then fill out the template below. Please read
+        our guide to filing a bug first: https://flutter.dev/docs/resources/bug-reports
```

Comment:
Let's remove this last sentence so that it's not duplicated with the new checkbox item below.

Reply:
Removed it